content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,95 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful realtime data synchronization feature designed to facilitate the synchronising of changes from backend databases to frontend clients. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably to all connected clients in realtime.
+
+LiveSync benefits applications requiring realtime collaboration, such as collaborative form editing, online Auction, task management applications, or simple multiplayer turn-based games, where maintaining data consistency and low latency are paramount. Developers can build highly responsive and interactive applications that deliver an exceptional realtime user experience.
+
+h2(#channel-based-broadcasting). Channel-based broadcasting
+
+The mechanism for synchronizing data updates across devices using Ably's LiveSync closely mirrors the delivery model of realtime messages to subscribers via Ably's "pub/sub channels":https://ably.com/docs/channels. When a message is published on an Ably channel, it is immediately broadcasted to all subscribers of that channel. Similarly, with LiveSync, data updates added to the outbox table are then published to all subscribed clients in realtime, ensuring seamless data synchronization between the database and all clients.
+
+h2(#frontend). Frontend
+
+Clients consume realtime updates to the database table through the Models SDK, which carries out the following steps:
```

Comment:
I think maybe this should be something like. "Clients consume realtime updates reflecting database table through....."